addons/website_event/views/event_templates_widgets.xml

```diff
@@ -5,12 +5,12 @@
 <template id="events_search_box_input" name="Events search box">
     <t t-call="website.website_search_box_input">
         <t t-set="_classes" t-valuef="o_wevent_event_searchbar_form flex-grow-1 #{_classes}"/>
-        <t t-set="search_type" t-valuef="events"/>
-        <t t-set="action" t-valuef="/event"/>
+        <t t-set="search_type" t-value="_search_type"/>
+        <t t-set="action" t-value="action"/>
         <t t-set="display_description" t-valuef="true"/>
         <t t-set="display_detail" t-valuef="false"/>
         <t t-set="search" t-value="search or searches and searches['search']"/>
-        <t t-set="placeholder">Search an event...</t>
+        <t t-set="placeholder" t-value="_placeholder"/>
         <t t-foreach="searches" t-as="item">
             <input t-if="item != 'search' and item_value != 'all'" type="hidden"
                 t-att-name="item" t-att-value="item_value"/>
```

Comment:
Sorry, it was probably not clear :smile: 

I checked again what the hidden inputs do in this context, and they're used to send the values of filter tags when you press "enter" in the website search bar. This keeps the same behavior as the previous implementation.

They are not considered for the suggestions though, as you did not include them in your domain when implementing the "_search_get_detail", but I'm guessing it's intended?

So this is why we need this extra bit of templating with the t-foreach, to include those inputs into the form submit. That's the reason why we have a "events_search_box_input" template, rather than calling "website_search_box_input" directly.

The part I'm not sure about is the `<t t-out="0"/>` here below, I'm pretty sure it's useless and can be removed, but it would need to be tested?

Let me know if this makes more sense.

Reply:
Hello @thsh-odoo 

You are absolutely right actually, we could call "website_search_box_input" and insert the foreach in there...
The only reason we'd make a separate template is to re-use it and lessen duplicated code, but it only seems used once?

If so, maybe we can get rid of it?

Sorry for the back and forth, just trying to do the cleaning now as we probably won't touch this code again after that for a long time :smile: 

Thank you for your patience & cheers!

Reply:
Hello @awa-odoo :wave: 
I have removed the custom search template and directly called the website template in 
track, event and exhibitor module 
But we can see some duplicate code :grimacing: 

Just waiting for the runbot :runner: 

Can you please review it again?

Thanks!! :smile: